composer.json

```diff
@@ -46,6 +46,7 @@
         "justinrainbow/json-schema": "^5.2.1",
         "phpdocumentor/reflection-docblock": "^3.0 || ^4.0 || ^5.1",
         "phpdocumentor/type-resolver": "^0.3 || ^0.4 || ^1.4",
+        "phpspec/prophecy": "^1.10",
```

Comment:
should not be needed

Reply:
Like I said, most of the existing PhpUnit tests were failing without this dependency, at least when I was testing MongoDB. They ran just fine after installing the package.
You can grep your `tests` directory for `use Prophecy` to get the picture.
If you still think I'm wrong, I can remove this requirement.

Reply:
I think that we fixed this by now as we're using Prophecy on other parts of the library.

Reply:
If you say so. :)